package.json

```diff
@@ -48,8 +48,10 @@
     "test": "cross-env CI=1 react-scripts test --env=jsdom",
     "test:watch": "react-scripts test --env=jsdom",
     "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache",
-    "build": "rollup -c",
-    "start": "rollup -c -w",
+    "clean": "rm -rf dist",
```

Comment:
Just curious - was this necessary to get the build to work, or just an enhancement on the side?

Reply:
Just an enhancement on the side

Reply:
Should we use rimraf for this? so we enable non Unix contributors? 🙂